rskj-core/src/integrationTest/resources/integration-test-rskj.conf

```
@@ -0,0 +1,16 @@
+peer {
+
+    discovery = {
+
+        # if peer discovery is off
+        # the peer window will show
+        # only what retrieved by active
+        # peer [true/false]
+        enabled = true
```

```suggestion
        enabled = false
```
one more thing spotted.. probably in the `regtest` mode we don't need discovery to be `enabled`